lesson_03/homework_03.py

```diff
@@ -0,0 +1,75 @@
+"""
+This module processes a text excerpt from 'The Adventures of Tom Sawyer'.
+
+It performs various text manipulations and analyses, aimed at extracting
+and counting specific elements within the text.
+"""
+
+adventures_of_tom_sawyer = '''\
+Tom gave up the brush with reluctance in his .... face but alacrity
+in his heart. And while
+the late steamer
+'Big Missouri' worked ....
+and sweated
+in the sun,
+"the retired artist sat on a barrel in the .... shade close by, "
+    "dangled his legs."
+munched his apple, and planned the slaughter of more innocents.
+There was no lack of material;
+boys happened along every little while;
+they came to jeer, but .... remained to whitewash. ....
+"By the time Ben was fagged out, Tom had traded the next chance "
+     "to Billy Fisher for
+a kite, in good repair;
+and when he played
+out, Johnny Miller bought
+in for a dead rat and a string to swing it with—and so on, and so on,
+hour after hour. And when the middle of the afternoon came, from being a
+poor poverty, stricken boy in the .... morning, Tom was literally
+rolling in wealth.'''
+
+# Task 01
+adventures_of_tom_sawyer = adventures_of_tom_sawyer.replace('\n', ' ')
+
+# Task 02
+adventures_of_tom_sawyer = adventures_of_tom_sawyer.replace('....', ' ')
+
+# Task 03
+adventures_of_tom_sawyer = ' '.join(adventures_of_tom_sawyer.split())
+
+# Task 04
+h_count = adventures_of_tom_sawyer.lower().count('h')
+print(f'Кількість літер "h": {h_count}')
+
+# Task 05
+capitalized_words = [
+    word for word in adventures_of_tom_sawyer.split() if word[0].isupper()
+]
+print(f'Кількість слів з Великої літери: {len(capitalized_words)}')
+
+# Task 06
+second_tom_position = (
+    adventures_of_tom_sawyer.lower().split().index('tom',
+    adventures_of_tom_sawyer.lower().split().index('tom') + 1)
+)
```

Comment:
```suggestion
second_tom_position = adventures_of_tom_sawyer.find('Tom', adventures_of_tom_sawyer.index('Tom')+1)
```
Your code looks too complicated and seems gives incorrect result. Just pay attention to find() call abilities 
```
Help on method_descriptor:

find(...)
    S.find(sub[, start[, end]]) -> int
    
    Return the lowest index in S where substring sub is found,
    such that sub is contained within S[start:end].  Optional
    arguments start and end are interpreted as in slice notation.
    
    Return -1 on failure 
```